### PR TITLE
Fix internal link ids (e.g. for MariaDB clusters with auto_increment_increment != 1)

### PR DESCRIPTION
**Pull request by psy-q**
This should take care of #4137. 

Apologies for the messy commits, if this is not OK I can try to somehow make it cleaner, though I'm not sure how right now (not sure how my earlier PR was merged and why my old commit still shows).

[x] I read the [contributor guidelines](https://docs.kanboard.org/en/latest/developer_guide/contributing.html#i-want-to-contribute-to-the-code)

### CODE REVIEW

File: app/Schema/Mysql.php [hunk 1]
```diff
@@ -1069,17 +1069,32 @@ function version_46(PDO $pdo)
...
-    $rq->execute(array('is duplicated by', 4));
-    $rq->execute(array('is a child of', 7));
-    $rq->execute(array('is a parent of', 6));
-    $rq->execute(array('targets milestone', 9));
-    $rq->execute(array('is a milestone of', 8));
-    $rq->execute(array('fixes', 11));
-    $rq->execute(array('is fixed by', 10));
+
+    $rq->execute(array('blocks', 0));
+    $rq->execute(array('is blocked by', get_last_insert_id($pdo)));
```
Comment (fguillot) [inline app/Schema/Mysql.php]: From where is coming the function `get_last_insert_id()`?

Comment (psy-q) [reply app/Schema/Mysql.php]: I've seen it used elsewhere: https://github.com/kanboard/kanboard/blob/master/app/Schema/Migration.php#L70

Comment (fguillot) [reply app/Schema/Mysql.php]: Ah ok. I don't even remember what I did :)

Give me some time to test your pull-request on a Galera cluster.

Comment (psy-q) [reply app/Schema/Mysql.php]: Great, thanks a lot! We tested it on an MMM cluster, but Galera is better, it automatically changes the `auto_increment_increment` as nodes join and leave the cluster.

**Pull request merged**